lib/browser/webapis-media-query.ts

```diff
@@ -23,16 +22,14 @@ import {NestedEventListenerOrEventListenerObject, patchEventTargetMethods} from
             handler: args[0],
             target: self || _global,
             name: 'mediaQuery',
-            invokeAddFunc: function(
-                addFnSymbol: any, delegate: Task|NestedEventListenerOrEventListenerObject) {
+            invokeAddFunc: function(addFnSymbol: any, delegate) {
```

Comment:
Just like the review here,https://github.com/angular/zone.js/pull/631#pullrequestreview-21679511